web/src/compose_validate.ts

```diff
@@ -758,6 +813,11 @@ export function check_overflow_text($container: JQuery): number {
     const remaining_characters = max_length - text.length;
     const $indicator = $container.find(".message-limit-indicator");
     const is_edit_container = $textarea.closest(".message_row").length > 0;
+    if (!is_edit_container && text.length === 0) {
+        set_no_message_content(true);
+        return 0;
+    }
+    set_no_message_content(false);
```

Comment:
Don't do this check here. This function is also called `check_overflow_text` and checking for no message content is different than overflow text, which would be confusing.

But also this function is called when typing every character in the composebox and we only need to calculate this when we run the validate functions